test/arithmetic.jl

```diff
@@ -56,26 +56,26 @@ fast = UnivariateFinite(L, P, pool=missing);
     @test pdf(slow + slow, L) == pdf(fast + fast, L)
     t_slow = @belapsed1 $slow + $slow
     t_fast = @belapsed1 $fast + $fast
-    @test t_slow/t_fast > 10
+    @test t_slow/t_fast > 5
 
     @test pdf(slow - slow, L) == pdf(fast - fast, L)
     t_slow = @belapsed1 $slow - $slow
     t_fast = @belapsed1 $fast - $fast
-    @test t_slow/t_fast > 10
+    @test t_slow/t_fast > 5
 
     @test pdf(42*slow, L) == pdf(42*fast, L)
     @test pdf(slow*42, L) == pdf(fast*42, L)
     t_slow = @belapsed1 42*$slow
     t_fast = @belapsed1 42*$fast
-    @test t_slow/t_fast > 10
+    @test t_slow/t_fast > 5
     t_slow = @belapsed1 $slow*42
     t_fast = @belapsed1 $fast*42
-    @test t_slow/t_fast > 10
+    @test t_slow/t_fast > 5
 
     @test pdf(slow/42, L) == pdf(fast/42, L)
     t_slow = @belapsed1 $slow/42
     t_fast = @belapsed1 $fast/42
-    @test t_slow/t_fast > 10
+    @test t_slow/t_fast > 5
```

Comment:
I see that you changes some values her. Was it that the changes made things slower or the slow implementation got slightly faster.

Reply:
For reasons I did not investigate, on later versions of Julia, there was a deterioration in the speedup on Windows machines. Since this is just a smoke test to make sure our optimisations are indeed kicking in, I wasn't too fussed about relaxing this bound.